js/components/controlBar.js

```diff
@@ -317,6 +317,26 @@ var ControlBar = createReactClass({
     this.props.controller.toggleMoreOptionsScreen(this.moreOptionsItems);
   },
 
+  filterAudioControls: function() {
+    //We receive location param in desktopContent, instead of audioOnly.
+    //This is necessary to display or not the button, depending on Backlot settings.
+    var defaultConfig = this.props.skinConfig.buttons.audioOnly;
```

Comment:
To avoid unintentional change of the original config use `JSON.parse(JSON.stringify())` object clone please.